packages/kbn-esql-validation-autocomplete/src/autocomplete/autocomplete.test.ts

```diff
@@ -37,6 +37,7 @@ const ESQL_NUMERIC_TYPES = ESQL_NUMBER_TYPES as unknown as string[];
 const ESQL_COMMON_NUMERIC_TYPES =
   UNCASTED_ESQL_COMMON_NUMERIC_TYPES as unknown as FunctionReturnType[];
 
+const commandDefinitions = unmodifiedCommandDefinitions.filter((r) => r.name !== 'inlinestats');
```

Comment:
Good one, I guess we'll do exactly the same for https://github.com/elastic/kibana/issues/189945

Reply:
++